add single_cell_citeseq_bcrtcr_manifest_template.csv

chaodi51 opened the pull request:
This is to close https://github.com/childrens-bti/internal-ticket-tracker/issues/41

bicnasong reviewed (commented):
I see that `library_type` and `feature_type` have been added to the single-cell manifest template. I would suggest that wee keep a single unified manifest template for all single-cell modalities, which should make processing easier and avoid confusion about which template to use.

The only additional field I’d recommend is `chain_type` for TCR/BCR libraries, to identify which immune receptor chain the library captures. Acceptable values would be:

- **T-cell receptor chains**: `TCRα`, `TCRβ`
- **B-cell receptor chains**: `IGH`, `IGK`, `IGL`

This way, we have a single consistent manifest format while still capturing all necessary modality-specific information.

chaodi51 commented:
> I see that `library_type` and `feature_type` have been added to the single-cell manifest template. I would suggest that wee keep a single unified manifest template for all single-cell modalities, which should make processing easier and avoid confusion about which template to use.
> 
Make sense, I'll remove the earlier version. 

> The only additional field I’d recommend is `chain_type` for TCR/BCR libraries, to identify which immune receptor chain the library captures. Acceptable values would be:
> 
> * **T-cell receptor chains**: `TCRα`, `TCRβ`
> * **B-cell receptor chains**: `IGH`, `IGK`, `IGL`
> 
Added `chain_type` to [single_cell_citeseq_bcrtcr_manifest_template](https://childrensnational.box.com/s/r4lzvwfm6qm6zhfwyz0wsbluyg29d7ck), currently at `Box/Rokita-Lab/projects/scRNA-Seq/`. Let me know if it should be put elsewhere. @jharenza 

> This way, we have a single consistent manifest format while still capturing all necessary modality-specific information.
Updated this PR.

jharenza commented:
> I see that `library_type` and `feature_type` have been added to the single-cell manifest template. I would suggest that wee keep a single unified manifest template for all single-cell modalities, which should make processing easier and avoid confusion about which template to use.
> 
> The only additional field I’d recommend is `chain_type` for TCR/BCR libraries, to identify which immune receptor chain the library captures. Acceptable values would be:
> 
> * **T-cell receptor chains**: `TCRα`, `TCRβ`
> * **B-cell receptor chains**: `IGH`, `IGK`, `IGL`
> 
> This way, we have a single consistent manifest format while still capturing all necessary modality-specific information.

@bicnasong can you create a new issue here, referencing this to add that if experimental strategy is one of the above, then we need to have one of those above in the validator? @sickler-alex can work on that
